Application/Telegram/Helpers/IdentifyUserLocation.cs

```diff
@@ -36,6 +36,9 @@ public async Task<string> FindCountryAsync(double longitude, double latitude)
             _client.DefaultRequestHeaders.Accept.Clear();
             _client.DefaultRequestHeaders.Accept.Add(
                     new MediaTypeWithQualityHeaderValue("application/json"));
+
+            _client.DefaultRequestHeaders.Add("Accept-Language", "en");
```

Comment:
the results that was returned from api was language agnostic so I needed to change it to english only so we can easily filter the country.
Let's say if I send Warsaw's latitude and longitude the api returned Polska so after this change it will return Poland